tests/integration/Type/TypeFixture.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Commercetools\Core\IntegrationTests\Type;
+
+use Commercetools\Core\Client\ApiClient;
+use Commercetools\Core\Helper\Uuid;
+use Commercetools\Core\IntegrationTests\ResourceFixture;
+use Commercetools\Core\Model\Common\LocalizedString;
+use Commercetools\Core\Model\Type\FieldDefinition;
+use Commercetools\Core\Model\Type\FieldDefinitionCollection;
+use Commercetools\Core\Model\Type\StringType;
+use Commercetools\Core\Model\Type\Type;
+use Commercetools\Core\Model\Type\TypeDraft;
+use Commercetools\Core\Request\Types\TypeCreateRequest;
+use Commercetools\Core\Request\Types\TypeDeleteRequest;
+
+class TypeFixture extends ResourceFixture
```

Comment:
this class is in PR https://github.com/commercetools/commercetools-php-sdk/pull/524